src/types.ts

```diff
@@ -77,6 +78,20 @@ export type OrganizationModalAttributes = {
 	};
 };
 
+export type ActivityModalAttributes = {
+	type: Modal.ACTIVITY;
+	prefill?: {
+		subject?: string;
+		dueDate?: string;
+		dueTime?: string;
+		duration?: string;
+		note?: string;
+		description?: string;
```

Comment:
Maybe having this as `publicDescription` would be better to have consistency with what the modal expects from us?
```suggestion
		publicDescription?: string;
```